lib/protocol/auth/JWT.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 SAP AG.
```

Comment:
I believe this should be 'SAP SE'

Reply:
Ah, Copy&Paste error. The other files still have SAP AG as they where before the change from AG to SE. I fixed it for this file.